docs/build/sdks/js/zksync-ethers/providers.md

```diff
@@ -146,9 +146,11 @@ console.log(`Gas for token approval tx: ${gasTokenApprove}`);
 ```ts
 import { Provider, types } from "zksync-ethers";
 
+const ADDRESS = "<ADDRESS>";
+const RECEIVER = "<RECEIVER>";
 const provider = Provider.getDefaultProvider(types.Network.Sepolia);
-const tokenAddress = "0x765F5AF819D818a8e8ee6ff63D8d0e8056DBE150"; // Crown token which can be minted for free
-const paymasterAddress = "0x3cb2b87d10ac01736a65688f3e0fb1b070b3eea3"; // Paymaster for Crown token
+const tokenAddress = "0xA70dF8446A6AeA0017D60e97e816e141aa28759b"; // Crown token which can be minted for free
+const paymasterAddress = "0x57F48f0d845E0ed7C9Bf066cEbFF64FbeBE6AFEF"; // Paymaster for Crown token
```

Comment:
Are these correct for Sepolia?

Reply:
Yes, they are.
- [0xA70dF8446A6AeA0017D60e97e816e141aa28759b](https://sepolia.explorer.zksync.io/address/0xA70dF8446A6AeA0017D60e97e816e141aa28759b)
- [0x57F48f0d845E0ed7C9Bf066cEbFF64FbeBE6AFEF](https://sepolia.explorer.zksync.io/address/0x57F48f0d845E0ed7C9Bf066cEbFF64FbeBE6AFEF)

Reply:
These contracts are not verified, and thus shouldn't be included in our docs. If we can verify them, then let's include them.

Reply:
Here are the verified token and paymaster contracts (compiled with latest compiler):

- Token: [0x927488F48ffbc32112F1fF721759649A89721F8F](https://sepolia.explorer.zksync.io/address/0x927488F48ffbc32112F1fF721759649A89721F8F).
- Paymaster: [0x13D0D8550769f59aa241a41897D4859c87f7Dd46](https://sepolia.explorer.zksync.io/address/0x13D0D8550769f59aa241a41897D4859c87f7Dd46).